app/build.gradle

```diff
@@ -99,6 +99,12 @@ dependencies {
     implementation "com.google.android.gms:play-services-oss-licenses:$rootProject.oss_licenses"
     implementation "com.isseiaoki:simplecropview:$rootProject.cropviewVersion"
 
+    //Biometrics
+    implementation 'androidx.biometric:biometric:1.2.0-alpha05'
+
+    //Country Code picker
+    implementation 'com.hbb20:ccp:2.5.1'
```

Comment:
Add the version code inside the `ext block` in the `build.gradle (project)` file

Reply:
Done the changes in my local system. Should i push the changes keeping in mind the commit style and PR guidelines?

Reply:
Sure... Go ahead please